tests/functional/spec/components/radio/radio.spec.js

```diff
@@ -0,0 +1,163 @@
+const helpers = require('../../../helpers');
+
+describe('Component: Radio', function() {
+  describe('Given I start a Mandatory Radio survey', function() {
+
+    var RadioMandatoryPage = require('../../../pages/components/radio/mandatory/radio-mandatory.page');
+    var RadioSummaryPage = require('../../../pages/components/radio/mandatory/summary.page');
+
+    var radio_schema = 'test_radio_mandatory.json';
```

Comment:
Why is the 1st schema specified as a variable then subsequent ones are inline. we need to make the pattern consistent